workflows/sample/tasks/deepvariant_round1.wdl

```diff
@@ -19,14 +22,17 @@ task make_examples_round1 {
     String commands_name = "commands.txt"
   }
 
+#  Float multiplier = 3.25
+#  Int disk_size = ceil(multiplier * (size(reference.datafile, "GB") + size(reference.indexfile, "GB") + size(bams, "GB") + size(bais, "GB"))) + 20
+  Int disk_size = 500
+
   command <<<
+    echo requested disk_size =  ~{disk_size}
+
     make_examples_func="     /opt/deepvariant/bin/make_examples \
         --norealign_reads \
         --vsc_min_fraction_indels ~{vsc_min_fraction_indels} \
         --alt_aligned_pileup=diff_channels \
-        --add_hp_channel \
-        --sort_by_haplotypes \
-        --parse_sam_aux_fields \
```

Comment:
These params were added to support the change to DeepVariant v1.1.

Reply:
And the same params need to be present in make_examples_round2.

Reply:
Parameters added in make_examples, rounds 1 and 2